src/components/SpatialViewer/SpatialViewer.js

```diff
@@ -1,101 +0,0 @@
-import React, { Component } from 'react';

```

Comment:
Removing spatial viewer as it isn't needed for the repository, and the vitessce package has a number of vulnerabilities associated with it.